ansible/collections/bento/common/roles/opensearch_dataloading/files/opensearch-backup.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import boto3
+import requests
+from requests_aws4auth import AWS4Auth
+
+parser = argparse.ArgumentParser(description='Opensearch Backup Script')
+parser.add_argument("--oshost", type=str, help="opensearch host with trailing /")
+parser.add_argument("--repo", type=str, help="opensearch snapshot repository")
+parser.add_argument("--s3bucket", type=str, help="s3 bucket")
+parser.add_argument("--basePath", type=str, help="s3 bucket base path")
+parser.add_argument("--snapshot", type=str, help="opensearch snapshot value")
+parser.add_argument("--rolearn", type=str, help="role arn - typically power user role")
+args = parser.parse_args()
+print(args)
+oshost = args.oshost
+repo = args.repo
+
+s3bucket= args.s3bucket
+snapshot= args.snapshot 
+basepath= args.basePath
+rolearn= args.rolearn
+
+host = (oshost) 
+region = 'us-east-1' 
```

Comment:
can the region be passed in - we always use the same region right now, but that may not always be the case

Reply:
region = boto3.Session().region_name